sw/device/silicon_creator/rom/rom.c

```diff
@@ -672,11 +672,27 @@ void rom_interrupt_handler(void) {
   OT_UNREACHABLE();
 }
 
+void rom_exception_handler(void) {
+  uint32_t mcause;
+  CSR_READ(CSR_REG_MCAUSE, &mcause);
+  ibex_exception_code_t exception_code =
+      (ibex_exception_code_t)(mcause & kIbexExceptionCodeMax);
```

Comment:
Should Max be name Mask instead?

Reply:
It's both the Max exception code _and_ the Mask.  And moot, since the handler is now in assembly language :)